src/ZODB/MappingStorage.py

```diff
@@ -232,7 +233,7 @@ def sortKey(self):
 
     # ZODB.interfaces.IStorage
     @ZODB.utils.locked(opened)
-    def store(self, oid, serial, data, version, transaction):
+    def store(self, oid, serial, data, version, transaction, resolved=False):
```

Comment:
Please no the ``resolved`` argument. :)  It couples ``MappingStorage`` and ``DemoStorage`` too tightly.

Also, one of my goals for ``MappingStorage`` was that it would provide an illustration of how the storage API might be implemented.

``DemoStorage`` does conflict resolution.  ``MappingStorage`` doesn't and should have ``_resolved``.